### ISSUE TITLE
Can we use the lognorm function in scipy?

### ISSUE THREAD

**Issue by ruixue-li** [UC-MACSS/persp-model_W18#113]
I'm just wondering do we need to compute the pdf of lognorm on our own or can we use scipy.stats.lognorm.pdf/cdf?

**Comment by Otamio** [comment]
Maybe it's too late.
I noticed that problem too when I was doing PS2. There may be two cons of computing the pdf using the original formula: (1) error prone; (2) may encounter problems when x=0. Therefore I simply used scipy.stats.lognorm.pdf/cdf. That works perfectly fine for me.

**Comment by rickecon** [comment]
Yes. The lognorm function works great. You just have to make sure you have the right location and scale parameter. You can also calculate the mean E[X] and variance Var[X] analytically as functions of mu and sigma.
